facebook-htb.js

```diff
@@ -117,7 +118,9 @@ function FacebookHtb(configs) {
             sdk: '5.5.web',
             placementids: [],
             adformats: [],
-            cb: System.generateUniqueId(56, 'ALPHANUM')  // cachebuster
+            cb: System.generateUniqueId(56, 'ALPHANUM'), // cachebuster
+            w: 'Index Exchange',
+            adv: adapterVersion
```

Comment:
I'd love to log the wrapper version as well, not just the adaptor version. Would you be able to add something to the Utility Library letting me query version, please?